docs/changelog.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+### 03-February-2022 - 17:01 CET
+
+- [feature] New label to identify pull-requests that are just bumping dependencies version.
```

Comment:
🤯

Reply:
I'd like to update my bot but https://github.com/conan-io/conan-center-index/labels does not have anything "new" 😕 

What's the new label?

Reply:
Good catch. It's `Bump dependencies` (I added now): https://github.com/conan-io/conan-center-index/labels/Bump%20dependencies

It was not listed on github labels, but still the github api allows adding a label to PR's which is not listed in labels.